inc/indexables/class-term-indexable.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Internals
```

Comment:
Internals?

Reply:
I'm wondering too why this is in "Internals". A new "Indexables" package might fit better.